src/components/TasksList/TasksList.tsx

```diff
@@ -34,7 +35,7 @@ export const TasksList = ({ pageSize = 10 }: { pageSize?: number }) => {
       {sortedTasks && sortedTasks.length > 0 ? (
         <>
           {sortedTasks.map((task, index) => (
-            <TasksListItem key={index} task={task} index={index} />
+            <TasksListItem key={task.id} task={task} index={index} />
```

Comment:
The ordering changes when we submit a task (which is good) but causes classic react bugs where the value of text in each box shifts when you submit.